Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -978,16 +991,17 @@ public static void DestroyPlayerForConnection(NetworkConnection conn)
         /// <para>This will cause a new object to be instantiated from the registered prefab, or from a custom spawn function.</para>
         /// </summary>
         /// <param name="obj">Game object with NetworkIdentity to spawn.</param>
+        /// <param name="client">Client associated to the object.</param>
         /// <param name="ownerConnection">The connection that has authority over the object</param>
-        public static void Spawn(GameObject obj, NetworkConnection ownerConnection = null)
+        public void Spawn(GameObject obj, NetworkClient client, NetworkConnection ownerConnection = null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/manual/paulpach/MirrorNG/pullRequest?prid=4841467)